website/index_old.html

```diff
@@ -22,15 +22,15 @@
     <meta name="msapplication-TileColor" content="#ffffff">
     <meta name="msapplication-config" content="resources/browserconfig.xml">
     <meta name="theme-color" content="#9a93c6">
-
+    <link rel="stylesheet" href="./styles/style.scss">
     <!-- TODO: Include Google Analytics -->
 </head>
 
 <body>
     <div class="container">
         <img src="resources/timer.svg" alt="App icon" width="150" />
         <h1>Time to Leave</h1>
-        <p>Log work hours and get notified when it's time to leave the office and start to live</p>
+        <p>Log work hours and get notified when it's time to leave the office and start to livee</p>
```

Comment:
```suggestion
        <p>Log work hours and get notified when it's time to leave the office and start to live</p>
```

Small typo, though I guess this isn't used anymore, is it? And likely could be deleted

Reply:
Thanks for pointing out the typo. I took the texts from the design.  @thamara might want to update the texts